core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -248,6 +248,8 @@
   public static final PropertyKey MASTER_LINEAGE_RECOMPUTE_LOG_PATH =
       create(Name.MASTER_LINEAGE_RECOMPUTE_LOG_PATH,
           String.format("${%s}/recompute.log", Name.LOGS_DIR));
+  public static final PropertyKey MASTER_LOST_FILE_DETECTOR_ENABLED =
```

Comment:
Do you want to make the default to be false?

Reply:
@yupeng9 Yeah, i really want to make the default to be false, be true just to keep consistency of the formers. i am glad you don't mind, i change to false right know.